web/styles/right_sidebar.css

```diff
@@ -517,13 +510,52 @@ $user_status_emoji_width: 24px;
     }
 }
 
-.right-sidebar-shortcuts {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
+.buddy-list-user-link,
+.invite-user-shortcut {
+    margin-right: var(--width-simplebar-scroll-hover);
+    padding-left: var(--right-sidebar-left-spacing);
+    border-radius: 4px;
+
+    &:hover {
+        background: var(--color-background-sidebar-action-heading-hover);
+        box-shadow: inset 0 0 0 1px var(--color-shadow-sidebar-row-hover);
+        color: var(--color-text-sidebar-action-heading-hover);
+    }
+
+    .buddy-list-user-link-text,
+    .invite-user-link {
+        display: grid;
+        grid-template-rows: var(--line-height-sidebar-row-prominent);
+        text-decoration: none;
+        align-items: center;
+        color: var(--color-text-sidebar-action-heading);
+
+        &:active {
```

Comment:
I don't think we've added these transitions to other similar elements, but a nice follow up would be to turn these into variables and apply them to SHOW ALL TOPICS, etc.

Reply:
Yeah I can do that! Good call

Reply:
That sounds worth doing if we like the transitions. Let's first have a #design thread specifically about these transitions; I'm not sure they're entirely helpful in this sidebar UI, and want folks to play with it and discuss before we invest more in using them. I feel like they're basically good on buttons but these elements feel more like links than buttons to me, so I'm not sure it's right to have them.